<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Decode</class>
 <widget class="QDialog" name="Decode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extract data from file</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="stegoTypeGroupBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
border: none;
font: 12px;
color: #fff;
}</string>
   </property>
   <property name="title">
    <string>Steganography Decode Type</string>
   </property>
   <widget class="QRadioButton" name="lsbRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>LSB</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="msbRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>MSB</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="randomBitRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Random bit</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="containerFileLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>130</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>stegoPicture.bmp</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>42</string>
   </property>
  </widget>
  <widget class="QLabel" name="keyLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Key for Random&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="containerLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Container File&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="musicBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
color: #fff;
font-size: 13px;
}</string>
   </property>
   <property name="text">
    <string>Music File</string>
   </property>
  </widget>
  <widget class="QLabel" name="aesKeyLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;AES256 Key&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="aesKeyLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>260</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1234567890qazwsxedcrfvtgbyhnujmi</string>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>240</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
color: #fff;
}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
